libusb/Makefile.haiku

```diff
@@ -0,0 +1,46 @@
+###########################################
+# Simple Makefile for HIDAPI test program
+#
+# Alan Ott
+# Signal 11 Software
+# 2010-06-01
+###########################################
+
+all: hidtest libs
+
+libs: libhidapi.so
+
+CC       ?= cc
+CFLAGS   ?= -Wall -g -fPIC
+
+CXX      ?= c++
+CXXFLAGS ?= -Wall -g
+
+COBJS     = hid.o
+CPPOBJS   = ../hidtest/hidtest.o
```

Comment:
Please adjust the manual Makefile as in https://github.com/libusb/hidapi/commit/51bdec78448c7c5e5c16d41e56452befee19ce7f